CONTRIBUTING.md

```diff
@@ -67,9 +67,9 @@ docker run apache/kibble-dev pytest
 ## Testing
 
 Apache Kibble project uses [pytest](https://docs.pytest.org/en/stable/) for running testing. Writing
-good test helps us avoid regression and unexpected issues.
+the good test helps us avoid regression and unexpected issues.
```

Comment:
Should be 'Writing a good test.. or *Writing good tests helps ....'

Reply:
Thank you for the review @sharanf , I modified based on your review.

"Writing good tests help us avoid regression and unexpected issues."